2-projects/1-todo-list/script.js

```diff
@@ -1,25 +1,77 @@
-function populateTodoList(todos) {
+function populateTodoList(todoElement) {
   let list = document.getElementById("todo-list");
+  list.style.backgroundColor = "white";
+
+  let listItem = document.createElement("li");
+  list.appendChild(listItem);
+
```

Comment:
Nice work! We can currently add empty todos, can you think of a way to prevent this happening?

It might also be nice to our to provide a cursor style for the delete / completed icons (like when we hover over the 'Add Todo' or 'Remove all completed' buttons